database/schema.sql

```diff
@@ -3,29 +3,33 @@ CREATE DATABASE sdcphotogallery;
 
 \c sdcphotogallery;
 
-CREATE TABLE users(
-id BIGSERIAL NOT NULL PRIMARY KEY,
-email varchar(30) NOT NULL,
+CREATE TABLE IF NOT EXISTS users(
+id SERIAL NOT NULL PRIMARY KEY,
+email varchar(50) NOT NULL,
 first_name varchar(20) NOT NULL,
-last_name varchar(20) NOT NULL,
+last_name varchar(20) NOT NULL
 );
 
-CREATE TABLE rooms (
-id BIGSERIAL NOT NULL PRIMARY KEY,
-user_Id BIGINT REFERENCES users(id), --not sure if necessary (owner owns rooms?)
-title varchar (30) NOT NULL,
-address varchar (70) NOT NULL,
+CREATE TABLE IF NOT EXISTS rooms (
+id SERIAL NOT NULL PRIMARY KEY,
+title varchar (80) NOT NULL,
```

Comment:
I had some problems with titles generated by faker, some of them can be really long, why not just using "text" instead of varchar(limit), it has same utf-8 but no need to worry about length. Though you've probably made it for security reasons, but in terms of sdc I don't think anybody will access your DB and store something in one of your tables.